ci/docs/hypershift_setup.md

```diff
@@ -0,0 +1,79 @@
+# Hypershift
+
+HyperShift is middleware for hosting OpenShift control planes at scale that solves for cost and time to provision, as well as portability across cloud service providers with strong separation of concerns between management and workloads. Clusters are fully compliant OpenShift Container Platform (OCP) clusters and are compatible with standard OCP and Kubernetes toolchains.
+
+# Steps to setup Hypershift on ROSA
+1. Login to the ROSA cluster created using [ROSA Provisioning](https://github.com/openshift-pipelines/pipeline-service/tree/main/ci/docs/rosa_cluster_provision.md) documentation.
+```bash
+oc login <API_URL>  -u <USERNAME> -p <PASSWORD>
+```
+2. Create MultiClusterEngine to enable Hypershift
+```yaml
+apiVersion: multicluster.openshift.io/v1
```

Comment:
Instead of documenting the manifests, maybe they should leave in the repository (maybe in `ci/manifests/hypershift`)?